spec/models/user_spec.rb

```diff
@@ -0,0 +1,133 @@
+
+  scenario "User adds item successfully" do
+    visit '/'
+    click_button('Post a Series')
+    fill_in('Title') with: title_one.title
+    fill_in('Media Type') with: title_one.media_type
+    fill_in('Description') with: title_one.description
+    fill_in('Link') with: title_one.link
+    expect(page).to have_content('EastWillyB')
+    expect(page).to have_content('Willie Jr.’s got his work cut out for him. Running Bushwick’s favorite Latino bar isn’t easy when the old timers retire, the new generation forgets their culture, and the hipsters start taking over the hood. As if it couldn’t get worse, Maggie, Willie Jr.’s ex-fiance, has taken up with Willie Jr.’s arch-nemesis, Albert, and is skillfully transforming Albert’s competing neighborhood bar into a moneymaker, as they cater to the whims of the new residents. Along with the help of his bar regulars, Willie Jr. tries to come up with a solution quickly, or risk losing the two loves of his life: Maggie and his sports bar.')
+  end
+
+  scenario "User adds item unsuccessfully" do
+    visit '/'
+    click_button('Post a Series')
+    fill_in('Title') with: title_one.title
+    fill_in('Media Type') with: title_one.media_type
+    fill_in('Description') with: title_one.description
+
+    expect(page).to_not have('EastWillyB')
+    expect(page).to_not have('Willie Jr.’s got his work cut out for him. Running Bushwick’s favorite Latino bar isn’t easy when the old timers retire, the new generation forgets their culture, and the hipsters start taking over the hood. As if it couldn’t get worse, Maggie, Willie Jr.’s ex-fiance, has taken up with Willie Jr.’s arch-nemesis, Albert, and is skillfully transforming Albert’s competing neighborhood bar into a moneymaker, as they cater to the whims of the new residents. Along with the help of his bar regulars, Willie Jr. tries to come up with a solution quickly, or risk losing the two loves of his life: Maggie and his sports bar.')
+  end
+
+  scenario "User adds item unsuccessfully" do
+    visit '/'
+    click_button('Post a Series')
+    fill_in('Media Type') with: title_one.media_type
+    fill_in('Description') with: title_one.description
+    fill_in('Link') with: title_one.link
+
+    expect(page).to_not have('http://www.eastwillyb.com/')
+    expect(page).to_not have('Willie Jr.’s got his work cut out for him. Running Bushwick’s favorite Latino bar isn’t easy when the old timers retire, the new generation forgets their culture, and the hipsters start taking over the hood. As if it couldn’t get worse, Maggie, Willie Jr.’s ex-fiance, has taken up with Willie Jr.’s arch-nemesis, Albert, and is skillfully transforming Albert’s competing neighborhood bar into a moneymaker, as they cater to the whims of the new residents. Along with the help of his bar regulars, Willie Jr. tries to come up with a solution quickly, or risk losing the two loves of his life: Maggie and his sports bar.')
```

Comment:
what's the story with these "huge" strings in the test files? Is it "necessary" to have these strings be this big? they make it hard to really read these tests well/easily.